storage/rocksdb/ha_rocksdb.cc

```diff
@@ -5018,6 +5115,9 @@ int ha_rocksdb::open(const char *const name, int mode, uint test_if_locked) {
     DBUG_RETURN(HA_ERR_OUT_OF_MEM);
   }
 
+  m_ttl_bytes =
```

Comment:
ROCKSDB_SIZEOF_TTL_RECORD is 8 bytes. The size of the m_ttl_bytes is also 8 bytes. Why not just store ttl_bytes directly in m_ttl_bytes instead of allocating an array and having m_ttl_bytes point to it?